github/MainClass.py

```diff
@@ -789,6 +790,42 @@ def get_app(self, slug=github.GithubObject.NotSet):
             headers, data = self.__requester.requestJsonAndCheck("GET", f"/apps/{slug}")
             return GithubApp.GithubApp(self.__requester, headers, data, completed=True)
 
+    def get_installation_by_id(self, installation_id):
```

Comment:
This is now implemented by `GithubInstallation.get_app_installation(installation_id)`.